src/contrib/persistence/Akka.Persistence.Sqlite.Tests/Bugfix4360Spec.cs

```diff
@@ -136,14 +136,10 @@ public void Should_recover_persistentactor_sqlite()
             ExpectTerminated(recoveryActor);
 
             // recreate the actor and recover
+            var recoveryActor2 = Sys.ActorOf(Props.Create(() => new RecoverActor(TestActor)), recoveryActor.Path.Name);
```

Comment:
Once the `Terminated` is received, this actor should be safe to recreate again right away.

If it's not - we should do a "double-tap" and get a second `ExpectTerminated` instead of an `AwaitAssert` in this case.